src/test/java/org/wso2/lsp4intellij/utils/UtilsTest.java

```diff
@@ -0,0 +1,55 @@
+package org.wso2.lsp4intellij.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.ArrayList;
+
+public class UtilsTest {
+
+  @Test
+  public void testConcatenateArrays() {
+    Assert.assertArrayEquals(new Object[]{new String[]{"a", "b"},
+            new String[]{"c", "d"}}, new Utils().concatenateArrays(
+                    new String[]{"a", "b"}, new String[]{"c", "d"}));
+  }
+
+  @Test
+  public void testParseArgsSingleQuote() {
+    Assert.assertArrayEquals(new String[] {"\'\'"}, Utils.parseArgs(
+            new String[]{"\'\'"}));
+  }
+
+  @Test
+  public void testParseArgsDoubleQuote() {
+    Assert.assertArrayEquals(new String[] {"\"\""}, Utils.parseArgs(
```

Comment:
I would also do some tests with null and/or empty strings :)

Reply:
There is no default in the case statement to handle empty strings so these would just be ignored and so i would argue that this is redundant.

Reply:
Right... I put that in the wrong line.. the methods `arrayToString()` and `stringToList() `have a null check :)

Also I am not sure if you are testing `arrayToString()` (I did a find on chrome and didn't find it mentioned)

Reply:
Ive added a null test for `stringToList()` but nothing is being done about `arrayToString()` as this method does not appear to work correctly and Freddy agrees.
I did just write a little not to the owner in the PR text comment to inform them of this

Reply:
Alright :)